dali/test/dali_test_single_op.h

```diff
@@ -182,6 +182,9 @@ class DALISingleOpTest : public DALITest {
     SetBatchSize(32);
   }
   inline void TearDown() override {
+    jpeg_decoded_.clear();
+    png_decoded_.clear();
+    tiff_decoded_.clear();
```

Comment:
Isn't destructor enough? Why do you clear them explicitly?

Reply:
Done